tests/utils/__init__.py

```diff
@@ -16,8 +16,6 @@
 def setup_moderation(models=[]):
     from moderation import moderation
 
-    moderation._registered_models = {}
```

Comment:
If `setup_moderation` clears the list of registered models, then all models that registered themselves - either via `auto_discover` or using the `ModeratedModel` base class - will be removed as well. Re-running `auto_discover` does *not* reload the modules (in fact, reloading doesn't work in newer Django versions at all, because Managers now have a deprecation meta class that changes the classes in a way that does not allow reloading) and therefore all tests that check if registration works fail if this line remains.